sycl/doc/extensions/proposed/sycl_ext_intel_fpga_kernel_interface_properties.asciidoc

```diff
@@ -86,23 +94,23 @@ using the mechanism defined in sycl_ext_oneapi_kernel_properties.
 ```c++
 namespace sycl::ext::intel::experimental {
 
-enum class streaming_interface_options_enum {
+enum class streaming_interface_options_enum : /* unspecified */ {
```

Comment:
Why mention that this class inherits from an unspecified base class?  Does this affect the user in any way?  It seems like an implementation detail to me but maybe I'm missing something.

Reply:
I checked the previous commits, and it is unclear that this was added as a response to a specific comment, so I am removing it.